public/app/features/alerting/unified/components/AlertLabels.tsx

```diff
@@ -28,7 +28,7 @@ export const AlertLabels = ({ labels, commonLabels = {}, size }: Props) => {
   const hasCommonLabels = commonLabelsCount > 0;
 
   return (
-    <div className={styles.wrapper} role="list" aria-label="Labels">
```

Comment:
I would still like an `aria-label` here, but the problem is that this is what is being announced as `h2`. I guess here are some things to think about:

- How many labels do we expect?
- Does just reading them out loud provide a good `h2` navigation experience?
- Is there a more generic name for a group than a set of labels?

Reply:
Also, this should have a testid instead.

Reply:
Just wanted to point out these are only announced as `h2` in the groups page :)

The number of labels is technically unbounded, and when grouping is disabled it will pick all labels to group by. Sadly we don't have any other definition other than "set of labels".

Reply:
We should probably allow passing an aria-describedby or similar so we can link to an existing element. E.g. in this case, there's "Labels" nearby, but nothing linking them together

![image](https://github.com/grafana/grafana/assets/4377076/1aaa9305-328a-477a-9963-8520cee531df)

----

![image](https://github.com/grafana/grafana/assets/4377076/31fe1be2-16db-4abe-aed1-41e51371ac32)

----

I would expect that the list of labels is described by the element that contains "Labels", as otherwise we're duplicating the description?

Reply:
> Just wanted to point out these are only announced as `h2` in the groups page :)

I think it only makes sense in that use case, although a more "simple" title for a group would probably be prefferable.

I think the idea with using `describedby` is good